src/ldf-client-ui.js

```diff
@@ -736,15 +727,21 @@ if (typeof global.process === 'undefined')
           return { type: type, value: datasource };
         }),
       };
+      // Add pre-defined prefixes to query
       var prefixesString = '';
       if (this.options.queryFormat === 'sparql') {
         for (var prefix in this.options.prefixes)
           prefixesString += 'PREFIX ' + prefix + ': <' + this.options.prefixes[prefix] + '>\n';
       }
-      let query = prefixesString + queryWithoutPrefixes;
+      let query = prefixesString + this.$queryTextsIndexed[this.options.queryFormat].val();
```

Comment:
Could you check if it's possible to pass these prefixes directly to the SparqlParser options?
I think there was an option for that.
If that's possible, we don't need the `prefixesString` anymore.